sunpy/data/_sample.py

```diff
@@ -22,35 +22,30 @@
 
 # the files should include necessary extensions
 _sample_data = {
-    # Do roll image first because it's the largest file.
```

Comment:
Why the reordering? This comment is still relevant (it starts the download first and then the other 4 slots normally download the rest of the files while this one is running?)

Reply:
I wanted it alphabetical

Reply:
I mean *sure*, but this one should still be at the top.

Reply:
Undone.